import-export-cli/impl/importAPI.go

```diff
@@ -145,38 +156,66 @@ func replaceEnvVariables(apiFilePath string) error {
 }
 
 // importAPI imports an API to the API manager
-func importAPI(endpoint, filePath, accessToken string, extraParams map[string]string, isOauth bool) error {
+func importAPI(endpoint, filePath, accessToken string, extraParams map[string]string, isOauth bool, dryRun bool,
+	apiLoggingCmdFormat string) error {
 	resp, err := ExecuteNewFileUploadRequest(endpoint, extraParams, "file",
 		filePath, accessToken, isOauth)
 	utils.Logf("Response : %v", resp)
 	if err != nil {
 		utils.Logln(utils.LogPrefixError, err)
 		return err
 	}
-	if resp.StatusCode() == http.StatusCreated || resp.StatusCode() == http.StatusOK {
-		// 201 Created or 200 OK
-		fmt.Println("Successfully imported API.")
-		return nil
+	if dryRun {
+		if resp.StatusCode() == http.StatusOK && resp.String() != "" {
+			// 200 OK
+			var data JSONData
+			err := json.Unmarshal([]byte(resp.String()), &data)
+			if err != nil {
+				fmt.Println("Error parsing JSON:", err)
```

Comment:
Does err go to the tool execution level? If that is the case, I think we should print this entire error message in the verbose mode.

Reply:
this will get logged in the console, will add a single line error and print the stack trace in verbose mode